webclient/src/variables.scss

```diff
@@ -38,66 +41,55 @@ $code-bg: #fcf2f2;
 
 // -- Overwriting
 $primary-color: $theme-accent;
+$secondary-color: #64a0c8;
 $body-font-color: #3b4351;
 $light-color: #fff;
+$dark-color: #303742;
 $bg-color: #f7f8f9;
 
 /* === Dark mode === */
 @if $theme == "dark" {
     // -- Custom vars
     $header-color: #1b1c1e;
-    $header-shadow-color: rgba(0,0,0,.4);
-    $autocomplete-box-shadow: 0 .05rem .2rem rgba(0, 0, 0, 0.6);
+    $header-shadow-color: rgba(0, 0, 0, 40%);
+    $autocomplete-box-shadow: 0 .05rem .2rem rgba(0, 0, 0, 60%);
     $autocomplete-comment-color: #777;
     $autocomplete-filter-bg: #232526;
     $autocomplete-filter-text: #999;
     $footer-color: #1b1c1e;
     $footer-setting-bg-disabled: #333;
-    $feedback-overlay-bg: rgba(0, 0, 0, 0.7);
-    $feedback-box-shadow: 0 .2rem .5rem rgba(0, 0, 0, 0.5);
-    $toast-btn-bg: #fff1;
-    $toast-btn-bg-hover: #fff3;
-    $toast-btn-bg-active: #0001;
+    $feedback-overlay-bg: rgba(0, 0, 0, 70%);
+    $feedback-box-shadow: 0 .2rem .5rem rgba(0, 0, 0, 50%);
+    $toast-btn-bg: rgba(255, 255, 255, 7%);
+    $toast-btn-bg-hover: rgba(255, 255, 255, 2%);
+    $toast-btn-bg-active: rgba(0, 0, 0, 7%);
     $card-highlighted-bg: #0e0e0e;
     $text-gray: #aaa;
-    $border-default: #444;
+    $border-color: #444;
     $border-light: #444;
-    $card-border: $border-default;
-    $card-shadow: 3px 3px 4px rgba(0,0,0,.2);
+    $card-border: $border-color;
+    $card-shadow: 3px 3px 4px rgba(0, 0, 0, 20%);
     $card-shadow-dark: $card-shadow;
     $image-loading-bg: #444;
     $container-loading-bg: #2a2b2f;
-
     $roomfinder-selected-bg: #282c35;
-
     $search-border: #353535;
     $search-border-hover: #444;
     $code-bg: #201e1e;
 
-    // -- not implemented anywhere yet
-    $link-color: #59b2ff;
-
     // -- Overwriting
     $primary-color: #59b2ff !global;
-    $primary-color-dark: lighten($primary-color, 3%) !global;
-    $primary-color-light: darken($primary-color, 3%) !global;
     $secondary-color: #213d55 !global;
-    $secondary-color-dark: lighten($secondary-color, 3%) !global;
-    $secondary-color-light: darken($secondary-color, 3%) !global;
-    $link-color: $primary-color !global;
-    $link-color-dark: lighten($link-color, 5%) !global;
-    $link-color-light: darken($link-color, 5%) !global;
     $body-font-color: #d2d7df !global;
-    $dark-color: #9aa0aa !global;//#3e396b;
+    $dark-color: #9aa0aa !global;// #3e396b;
 
     // Also sets body background
     $light-color: #17181a !global;
 
     //      #0d0e13,         #13141a (also footer), link colors: #59b2ff,
     //    .. 1px solid #e1e1e140 on border
-    $border-color: $border-default !global;
+    $border-color: $border-color !global;
```

Comment:
This statement makes no sense. `$border-color` is a variable of spectre.css, and `$border-default` was our own variable so far. We should at least make it more clear that this is not our own variable and move all `border-color` statements into the "Overwriting" sections

Reply:
```suggestion
    $border-color: $border-default !global;
```